### ISSUE TITLE
Colon in LDIP template truncates value

### ISSUE THREAD

**Issue by lcneves** [nextcloud/ldap_write_support#81]
### Steps to reproduce:

**LDIP template:**
```
dn: uid={UID},{BASE}
objectClass: inetOrgPerson
objectClass: posixAccount
uid: {UID}
displayName: {UID}
cn: {UID}
sn: {UID}
userPassword: {PWD}
mail: {UID}@example.com
labeledURI: maildir:/var/mail/example.com/{UID}/
uidNumber: 5000
gidNumber: 5000
homeDirectory: /var/mail/example.com/{UID}/
```

**Resulting LDAP:**
```
# testuser, example.com
dn: uid=testuser,dc=example,dc=com
objectClass: inetOrgPerson
objectClass: posixAccount
uid: testuser
displayName: testuser
sn: testuser
userPassword:: <Hash>
labeledURI: maildir
uidNumber: 5000
gidNumber: 5000
homeDirectory: /var/mail/example.com/testuser/
cn: Test User
jpegPhoto:: <Contents>
mail: testuser@example.com
```
The value for `labeledURI` is truncated at the index of the colon character.

**Comment by vivacarvajalito** [comment]
Hello, i create one pull request for resolve this problem in https://github.com/nextcloud/ldap_write_support/pull/189

**Comment by susnux** [comment]
Fixed with #189
